base_ubl/__manifest__.py

```diff
@@ -10,6 +10,6 @@
     "author": "Akretion,Onestein,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/edi",
     "depends": ["uom_unece", "account_tax_unece", "base_vat", "pdf_helper"],
-    "external_dependencies": {"python": ["PyPDF2"]},
+    "external_dependencies": {"python": ["PyPDF2<=1.28.4"]},
```

Comment:
```suggestion
    "external_dependencies": {"python": ["PyPDF2<2"]},
```

Reply:
But actually, what happens if we simply leave "PyPDF2". Since it is already installed, pip should not upgrade it.

Could you try that so I can see the test log ?